cmd/tracee-ebpf/internal/config/config.go

```diff
@@ -0,0 +1,80 @@
+package config
+
```

Comment:
I would rename the `internal` dir to `input`

Reply:
I think this should still stay in `internal` so it won't be able to be imported outside of `cmd/tracee-{}`. Would there be a reason we'd want it the other way?